docker/step/Dockerfile

```diff
@@ -31,7 +31,8 @@ RUN /bin/bash -c 'source find_fastest_resources.sh \
     shap==0.30.1 \
     plotille==3.7 \
     seaborn==0.9.0 \
-    dill==0.3.0"
+    dill==0.3.0 \
+    sklearn2pmml==0.56.0"
```

Comment:
The `sqlflow_models` wheel package has been distributed on `./build` as the following, can we just `pip install build/*.whl`?

https://github.com/sql-machine-learning/sqlflow/blob/754beefac16e4acfe1e5ea13bdc02ef27338d002/docker/dev/build.sh#L74-L76

Reply:
Yes，@lhw362950217 's PR would fix the problem.